build/yamls/base/security-crds.yml

```diff
@@ -0,0 +1,121 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: networkpolicies.security.antrea.tanzu.vmware.com
+spec:
+  group: security.antrea.tanzu.vmware.com
+  versions:
+    - name: v1beta1
+      served: true
+      storage: true
+  scope: Namespaced
+  names:
+    plural: networkpolicies
+    singular: networkpolicy
+    kind: NetworkPolicy
+    shortNames:
+      - anp
+  # Prune any unknown fields
+  preserveUnknownFields: false
+  validation:
+    openAPIV3Schema:
+      type: object
+      properties:
+        spec:
+          # Ensure that Spec.AppliedTo and Spec.Priority fields are set
+          required:
+            - appliedTo
+            - priority
+          type: object
+          properties:
+           priority:
+             type: number
+             format: float
+             # Ensure that Spec.Priority field is between 1 and 10000
+             minimum: 1.0
+             maximum: 10000.0
```

Comment:
intentionally keeping it float, because @antoninbas reminded me the use case where if priorities are auto generated because of policies created via UI(in future) we could limit the updates in priority of all policies by using float